util/stdio.h

```diff
@@ -0,0 +1,21 @@
+  
+#ifndef LIBUTIL_STDIO_H
+#define LIBUTIL_STDIO_H
+
+/* Include the real sys/file.h. We don't want to recreate all the stuff */
+/* it contains - just override flock */
```

Comment:
Looks like this comment needs to be adjusted since it still references sys/file.h and flock.